### PR TITLE
ci: fix a regression when parsing reference in the webui

### PR DESCRIPTION
**Pull request by samoht**
This was introduced by d1fc7b77

Signed-off-by: Thomas Gazagnaire <thomas@gazagnaire.org>

### CODE REVIEW

File: ci/src/cI_web_templates.ml [hunk 1]
```diff
@@ -89,7 +89,7 @@ let escape_ref path =
   Uri.pct_encode ~scheme:"http" (String.concat ~sep:"/" path)
 
 let unescape_ref s =
-  Uri.pct_encode ~scheme:"http" s |> (fun s -> String.cuts ~sep:"/" s)
+  Uri.pct_decode s |> (fun s -> String.cuts ~sep:"/" s)
```
Comment (talex5) [inline ci/src/cI_web_templates.ml]: Shouldn't we split on `/` before decoding?

Comment (samoht) [inline ci/src/cI_web_templates.ml]: You are of course right

Comment (samoht) [inline ci/src/cI_web_templates.ml]: Actually I am not sure to understand. We used to decode and the split in d1fc7b7, I'm just re-doing the same thing here.

**Comment by talex5** [comment]
Tested and works - LGTM.

**Pull request merged**